README.md

```diff
@@ -79,6 +59,22 @@ make manifests
 
 More information can be found via the [Kubebuilder Documentation](https://book.kubebuilder.io/introduction.html)
 
+### Test strategy
```

Comment:
Could you fix line:
**NOTE:** Run `make --help` for more information on all potential `make` targets
to:
**NOTE:** Run `make help` for more information on all potential `make` targets